### ISSUE TITLE
Platform Admin page not loading and pods not ready yet

### ISSUE THREAD

**Issue by sunsunny-hub** [ForgeRock/forgeops#673]
i am unable to get login UI prompts after loading "my-namespace.iam.example.com/platform" on browser, please find the screenshot of the same. Not sure whether my browser request is redircting to login-ui pod by Kubernetes ingress controller.
Also my 2 pods [amster and ldif-imposter] is not ready yet, please find the screenshot for the same.
![platformAdmin](https://user-images.githubusercontent.com/73718143/219352292-346978c9-6245-4cad-a273-f428c2ba8a34.png)
![pods](https://user-images.githubusercontent.com/73718143/219352304-f0206918-d2cd-4af1-8da4-c7ffa21b94b3.png)

**Comment by dgoldssfo** [comment]
Could you post the debug logs as described here? https://backstage.forgerock.com/docs/forgeops/7.2/troubleshooting/pods.html

**Comment by sunsunny-hub** [comment]
@dgoldssfo Thanks for addressing my issue.
PFA attached log file as per the suggested link doc in the above comment.
I have attached the complete log of generated logs in /tmp/forgeops/log.html file in a log.txt file please have a look and one is the output screenshot of ./debug-log command.

![pods_log](https://user-images.githubusercontent.com/73718143/220620970-fd062af7-3c1c-441c-923b-b657d6cc6ecb.png)
[pods_log.txt](https://github.com/ForgeRock/forgeops/files/10803748/pods_log.txt)

**Comment by dgoldssfo** [comment]
@sunsunny-hub Could you please attach the HTML file? (It's formatted in such a way that makes things easier to troubleshoot.) Thanks.

**Comment by sunsunny-hub** [comment]
@dgoldssfo I did try to attach the html file but I am getting an error while attaching the HTML file "We don't support that file type" looks like GitHub doesn't support HTML file type in attachments. So I have to change the extension of the HTML file to txt, please revert the file type to HTML at your end and have look at the HTML log file.

[logs.txt](https://github.com/ForgeRock/forgeops/files/10836476/logs.txt)

**Comment by dgoldssfo** [comment]
Thanks for posting the logs. I took a look at the admin-ui pod logs and found this message, which is unexpected:

failed to try resolving symlinks in path "/var/log/pods/my-namespace_admin-ui-6cc597bfdb-tcc8p_7b977ac5-ccdf-4cdb-ac7d-7ea9508b8255/admin-ui/6.log": lstat /var/log/pods/my-namespace_admin-ui-6cc597bfdb-tcc8p_7b977ac5-ccdf-4cdb-ac7d-7ea9508b8255/admin-ui/6.log: no such file or directory

On a healthy deployment, the logs for this pod should look similar to the logs for the login-ui and end-user-ui pods. We don't have any reports of the error your getting, and I haven't seen it myself previously. 

You're deploying from branch release/7.1-20221111, correct?

By any chance, are you running this on WSL? It's documented as unsupported, here: https://backstage.forgerock.com/docs/forgeops/7.1/cdk/setup-cdk.html

One thing you might try is to delete the admin-ui pod and let it restart. Then check the container logs and see if you're still getting the "failed to try resolving symlinks" error. If not, you might be able to log in at that point. You shouldn't have to do that, but occasionally there are timing glitches. This doesn't look like one to me, but you never know. Worth a try.

I'll pass the error you're getting up to our engineers and see if they might be able to lend some insight into the situation.

**Comment by sunsunny-hub** [comment]
@dgoldssfo Yes that is correct, I am using release/7.1-20221111 branch. I'll give it a try and see if it works, and update you. Also I would like to thank you for your help @dgoldssfo.

**Comment by sunsunny-hub** [comment]
@dgoldssfo I did try what you suggested, i deleted the admin-ui pod and let it restart. Now the logs seem to be good as this time it didn't shows "failed to try resolving symlinks" but the issue is still not resolved admin page is still not loading up. Attaching the HTML log file.

[log1.txt](https://github.com/ForgeRock/forgeops/files/10847137/log1.txt)

**Comment by dgoldssfo** [comment]
@sunsunny-hub Could you please confirm you are not doing this on WSL/WSL2, and also let me know the commands you used to start Minikube and to install the CDK? Thanks.

**Comment by sunsunny-hub** [comment]
@dgoldssfo not using WSL/WSL2, I am using the GCP custom image of ubuntu pro 2004 LTS which supports [nested virtualization](https://cloud.google.com/compute/docs/instances/nested-virtualization/enabling).

I followed the commands mentioned in the ForgeRock legacy feature [Minikube](https://backstage.forgerock.com/docs/forgeops/7.1/legacy/cdk/minikube/setup/overview.html)

Although I installed the latest version, not the exact same version mentioned in the third-party software listed in the above link.
PFA file of my history command, while i completed the setup of CDK using minikube, line 1 to 11 is the cmd for setting up chrome remote desktop so that i can access GCP VM graphically.

[CDK_setup_cmd.txt](https://github.com/ForgeRock/forgeops/files/10857733/CDK_setup_cmd.txt)

**Comment by dgoldssfo** [comment]
@sunsunny-hub Apologies, I have been away at a conference, where I came down ill. Also working on a release, so I'm just super busy.

I assume you meant 20.04, not 2004, right? Is it an older version of VirtualBox as well? You might try upgrading both of those if possible. Running a VM inside a VM could be the source of your issue. Kubernetes does not always work perfectly in complex virtualization environments like what you have.

**Comment by sunsunny-hub** [comment]
@dgoldssfo Apologies for long delay, I was working on some important stuffs. Yes it was 20.04 and My VM was on Google cloud not on virtualbox.

**Comment by dgoldssfo** [comment]
I'm sorry, I'm not sure why this isn't working. Your environment is pretty different from what we've spec'ed in our doc, and not really supported. I think I've gone about as far as I can trying to figure out what the problem is. I'm reasonably certain that if you recreate the situation in the environment we specify in the doc, it'll work. Apologies.

**Comment by sunsunny-hub** [comment]
Ohk, So i will try again below is my 2 senarios you can suggest me which is better to try on.

Scenario 1- I have windows laptop i can created VM on VMware or Oracle Vbox and start delpoyment of CDM or CDK into that created VM as per the documentation -- (https://backstage.forgerock.com/docs/forgeops/7.1/cdm/gke/setup/setup-env.html)

Scenario 2 - Which is more like above, creating a VM on Google cloud which support virtualization and following the same documentation steps for CDK or CDM deployment. (https://backstage.forgerock.com/docs/forgeops/7.1/cdm/gke/setup/setup-env.html)

Please suggest the best way to achieve the ForgeRock Identity Platform deployment

**Comment by dgoldssfo** [comment]
Either is OK. You'll need to make sure your Linux VM supports virtualization within virotualization so that you can install VirtualBox within the VM.

In scenario 1, you'd still need to install VirtualBox within your Linux VM, and then use the VirtualBox driver when you install Minikube.
